web_widget_pattern/README.rst

```diff
@@ -0,0 +1,62 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+==============
+Input patterns
+==============
+
+This module was written to allow you to define a regex on character fields a user's input is tested against.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. depend on this module
+#. define a key ``pattern`` in the field's options dictionary::
+
+       <field name="email" options="{'pattern': '[\S]+@[\S]+'}" />
```

Comment:
Well, I know what is a regular expression, but I don't remember always the syntax, so that link is very handy for me.